.vscode/settings.json

```diff
@@ -2,6 +2,5 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "editor.formatOnPaste": true,
-  "vitest.shellType": "terminal",
```

Comment:
removing this as its actually a machine concern rather than an overall need for the repo.